.travis.yml

```diff
@@ -30,6 +29,10 @@ matrix:
       env: RAILS=master
   allow_failures:
     - env: RAILS=master
+branches:
+  only:
+    - master
+    - /\A\d-\d-stable\z/
```

Comment:
Master and stable branches in the activeadmin/activeadmin repo are the only branches that need this setting, because all other branches have PRs. Without this setting, PRs for branches in the activeadmin/activeadmin repo cause Travis to run the test suite twice.

<img width="651" alt="screen shot 2015-12-24 at 3 00 37 pm" src="https://cloud.githubusercontent.com/assets/688886/11997879/366c8506-aa4f-11e5-95af-ee7c7db76f29.png">